tuning/sft_trainer.py

```diff
@@ -92,7 +92,11 @@ def train(
         tokenizer.add_special_tokens({
             "pad_token": "<pad>",
         })
+    else:
```

Comment:
I think the special tokens are getting added 2 times here. One here and another time at line [118](https://github.com/foundation-model-stack/fms-hf-tuning/pull/15/files#diff-7d8f6f8626340337d03a2f04fec2840d2ede32b0c7af2640eed9759af45bb3cfR118)

Reply:
yes, there is a TODO for cleanup, confirmed not an issue